java/core/src/main/java/co/worklytics/psoxy/gateway/impl/ApiDataRequestHandler.java

```diff
@@ -329,12 +340,9 @@ ProcessedContent sanitize(HttpEventRequest request, RequestUrls requestUrls, Pro
     }
 
 
-    void writeSideOutput(HttpEventRequest request, com.google.api.client.http.HttpResponse sourceApiResponse, ProcessedContent sanitizedContent) {
-        try {
+    void writeSideOutput(HttpEventRequest request, com.google.api.client.http.HttpResponse sourceApiResponse,
+                         ProcessedContent sanitizedContent) throws Output.WriteFailure {
             apiDataSideOutputSanitized.write(request, sanitizedContent);
```

Comment:
If this is now a one-liner I would just inline the call. It took me a while to realize that the output involved was `apiDataSideOutputSanitized` . Also the response is not used